dockerfiles/is/sudoers

```diff
@@ -0,0 +1,50 @@
+# ------------------------------------------------------------------------
```

Comment:
Same as above, once Pod Security Policies are applied sudoers file would not take effect. We might need to use the operating system user group for granting permissions to persistent volumes.